frontend/resourceadm/components/ResourcePageInputs/ResourceLanguageTextField.tsx

```
@@ -81,17 +81,20 @@ export const ResourceLanguageTextField = ({
-      };
-    }, {} as SupportedLanguage);
-  };
-  const onBlurField = () => {
-    onBlur(getTrimmedTranslations());
-  };
+  useEffect(() => {
+    if (JSON.stringify(value) !== JSON.stringify(translations)) {
+      const trimmedTranslations = Object.keys(translations).reduce(
+        (acc: SupportedLanguage, key) => {
+          return {
+            ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)